python/paddle/nn/layer/norm.py

```diff
@@ -28,13 +28,10 @@
 # TODO: define normalization api  
 
 import six
-#from ...fluid.dygraph.nn import InstanceNorm
 
-from ...fluid.dygraph import BatchNorm  #DEFINE_ALIAS
-#from ...fluid.dygraph import GroupNorm  #DEFINE_ALIAS
+from ...fluid.dygraph import BatchNorm  # noqa: F401
```

Comment:
`BatchNorm` is no longer needed, export `BatchNorm1d`- 3d instead

Reply:
done

Reply:
export BatchNorm1d- 3d, but keep BatchNorm also to compatible with old usage